### DISCUSSION TITLE
Bibliography, Authors, Publications page

### DISCUSSION THREAD

**Discussion by kaanaksit** [squidfunk/mkdocs-material#5531] [Q&A]
`mkdocs-material` is a fantastic development, and I am very thankful to the developers for enabling this project. I use `mkdocs-material` in my research group's webpage, [Computational Light Laboratory](https://complightlab.com). It serves its purpose in the way I dreamed of. I wonder if there is a way to create a publications page using a `bibtex` bibliography, just like in the `al-folio` project. This could help me automate publishing a list of papers on our [publications page](https://complightlab.com/publications/). Please see my personal website's [publications page](https://kaanaksit.com/publications/) to understand what I mean. That would be wonderful if such an extension for `mkdocs-material` could also automate the author highlighting just like `al-folio`.

Once again, thank you very much! -- a user and supporter!

**Comment by squidfunk** [comment]
Thanks for your words and for supporting the project! Have you tried [`mkdocs-bibtex`](https://pypi.org/project/mkdocs-bibtex/)?

**Comment by kaanaksit** [comment]
Thank you for your suggestion. I gave a quick try to `mkdocs-bibtex.` It renders a traditional bibliography, which is suitable for writing in general. However, I hope to have something similar to my website's [publication page](https://kaanaksit.com/publications/). Is there a way to achieve this with `mkdocs-bibtex` or any other extension? Thank you for your help and support!

**Reply by squidfunk** [reply]
We're maintaining Material for MkDocs here, not `mkdocs-bibtex` ☺️ I think it's a better idea to ask the authors of the plugin, and maybe even submit a PR to add the functionality!
